addon/components/g-map-infowindow.js

```diff
@@ -43,10 +44,30 @@ const GMapInfowindowComponent = Ember.Component.extend({
     }
   },
 
+  optionsChanged: observer('disableAutoPan', 'maxWidth', 'pixelOffset', function() {
+    run.once(this, 'setOptions');
+  }),
+
+  setOptions() {
+    const infowindow = this.get('infowindow');
+    const options = ['disableAutoPan', 'maxWidth', 'pixelOffset'];
```

Comment:
Ah, you're right, better use setZindex directly then (it should be slightly more efficient on Google's side than running setOptions). Ok noted.